syllabus/syllabus-app/src/webapp/syllabus/css/jqueryui-editable.css

```diff
@@ -194,12 +194,15 @@ a.editable-click.editable-disabled:hover {
 }
 
 /*see https://github.com/vitalets/x-editable/issues/139 */
-.form-horizontal .editable
-{ 
+.form-horizontal .editable {
     padding-top: 5px;
     display:inline-block;
 }
 
+div.ui-dialog-buttonset button.ui-button.ui-corner-all.ui-widget {
+	color: #454545 !important;
```

Comment:
Please don't use important if you can avoid it

Reply:
I'm just going to drop this commit. I don't have the time to investigate unravelling the SASS that is causing this. I thought that because the scope is so limited, it wouldn't be a problem.